localisation/00_wc_sad.csv

```diff
@@ -672,29 +672,29 @@ EVTOPTA_WCOE.11;The wait was worth it.;;Das Warten hat sich gelohnt.;;;;;;;;;;;x
+EVTDESC_WCCLS.10101;To master ways of Ashamane you have to gain her blessing. You will depart on a long journey to understand her and her ways.;;Um die Wege von Ashamane zu meistern, m�sst Ihr ihren Segen erlangen. Ihr werdet Euch auf eine lange Reise begeben, um sie und ihre Wege zu verstehen.;;;;;;;;;;;x
+EVTOPTA_WCCLS.10101;I will depart immediately.;;Ich werde mich umgehend auf den Weg machen.;;;;;;;;;;;x
+EVTOPTB_WCCLS.10101;I already bear her blessing.;;Ich trage bereits ihren Segen.;;;;;;;;;;;x
+EVTDESC_WCCLS.10102;You have gained blessing of Ashamane and will now face a final test. You will undergo a ritual to relive Ashamanes memories from War of the Ancients and face pitlord Ronokon.;;Ihr habt den Segen von Ashamane erhalten und werdet nun einer letzten Pr�fung unterzogen. Ihr werdet Euch einem Ritual unterziehen, um die Erinnerungen von Ashamane an den Krieg der Ahnen und an den Kampf gegen den Grubenlord Ronokon wieder aufleben zu lassen.;;;;;;;;;;;x
+EVTOPTA_WCCLS.10102;I will not fail her.;;Ich werde sie nicht entt�uschen.;;;;;;;;;;;x
+EVTDESC_WCCLS.10103;The ritual has begun. As the world swirls around you the vision blurs and you start to hear weird sounds. You see the monstrous pitlord before you. You feel that your body is extremely tired like you were fighting the demon for hours. After trying to reach him again and again you can barely stand as exhaustion starts to blur your eyesight. With your final ounce of strength you leap on your enemy in your final act of desperation. You can feel that his foul spear in your chest but it doesn't matter now because yout fangs are plunged deep into his neck. Annihilan tries to free himself with all his might but it's too late for him. Fires of fel explosion devour both of you and the world starts to swirl around as you leave your vision.;;Das Ritual hat begonnen. W�hrend die Welt um Euch herumwirbelt, verschwimmt die Sicht und Ihr beginnt, seltsame Ger�usche zu h�ren. Ihr seht den monstr�sen Grubenlord vor Euch. Ihr sp�rt, dass Euer K�rper extrem m�de ist, als h�ttet Ihr stundenlang gegen den D�mon gek�mpft. Nachdem Ihr wiederholt versucht habt, ihn zu erreichen, k�nnt Ihr kaum noch stehen, da die Ersch�pfung beginnt, Euer Augenlicht zu tr�ben. Mit dem letzten Qu�ntchen Kraft springt Ihr in einem letzten Akt der Verzweiflung dem Feind entgegen. Ihr sp�rt, dass sein fauliger Speer in Eurer Brust steckt, aber das ist jetzt egal, denn Eure Rei�z�hne sind tief in seinen Nacken eingedrungen. Der Annihilaner versucht mit aller Kraft, sich zu befreien, aber es ist zu sp�t f�r ihn. Die Feuer der Fel-Explosion verzehren euch beide, und die Welt beginnt umherzuschwirren, als Ihr die Vision verlasst.;;;;;;;;;;;x
```

Comment:
```suggestion
EVTDESC_WCCLS.10103;The ritual has begun. As the world swirls around you the vision blurs and you start to hear weird sounds. You see the monstrous pitlord before you. You feel that your body is extremely tired like you have been fighting the demon for hours. After trying to reach him again and again you can barely stand as exhaustion starts to blur your eyesight. With your final ounce of strength you leap on your enemy in your final act of desperation. You can feel his foul spear in your chest but it no longer matters because your fangs are plunged deep into his neck. Annihilan tries to free himself with all his might but it's too late for him. Fires of fel explosions devour both of you and the world starts to swirl around as you leave your vision.;;Das Ritual hat begonnen. W�hrend die Welt um Euch herumwirbelt, verschwimmt die Sicht und Ihr beginnt, seltsame Ger�usche zu h�ren. Ihr seht den monstr�sen Grubenlord vor Euch. Ihr sp�rt, dass Euer K�rper extrem m�de ist, als h�ttet Ihr stundenlang gegen den D�mon gek�mpft. Nachdem Ihr wiederholt versucht habt, ihn zu erreichen, k�nnt Ihr kaum noch stehen, da die Ersch�pfung beginnt, Euer Augenlicht zu tr�ben. Mit dem letzten Qu�ntchen Kraft springt Ihr in einem letzten Akt der Verzweiflung dem Feind entgegen. Ihr sp�rt, dass sein fauliger Speer in Eurer Brust steckt, aber das ist jetzt egal, denn Eure Rei�z�hne sind tief in seinen Nacken eingedrungen. Der Annihilaner versucht mit aller Kraft, sich zu befreien, aber es ist zu sp�t f�r ihn. Die Feuer der Fel-Explosion verzehren euch beide, und die Welt beginnt umherzuschwirren, als Ihr die Vision verlasst.;;;;;;;;;;;x
```

Reply:
done, plus a little more